test/adapters/net_http_persistent_test.rb

```diff
@@ -10,8 +10,10 @@ def setup
         if defined?(Net::HTTP::Persistent)
           # work around problems with mixed SSL certificates
           # https://github.com/drbrain/net-http-persistent/issues/45
-          http = Net::HTTP::Persistent.new('Faraday')
-          http.ssl_cleanup(4)
+          if Net::HTTP::Persistent::instance_methods.include?(:ssl_cleanup)
+            http = Net::HTTP::Persistent.new('Faraday')
+            http.ssl_cleanup(4)
+          end
```

Comment:
Ok so in order for tests to work properly, we need to set this to false at the beginning.
I'm wondering if this option exists on v2.9 as well given it was there in 2011 😮